tokencache.go

```diff
@@ -41,3 +50,33 @@ func (c *MemoryTokenCache) Get(key string) (*Token, bool) {
 
 	return token, ok
 }
+
+// StartGC starts garbage collection of expired tokens.
+func (c *MemoryTokenCache) StartGC(ctx context.Context, gcInterval time.Duration) {
```

Comment:
How would a user access this when they've constructed a Client and have started using it?

Reply:
Likewise, how do you stop this?

Reply:
You would have to start it when instantiating the cache. The context can be used to stop it.

The cache can be passed in as an option when the client is constructed.

```go
fnTokenCache := sdk.NewMemoryTokenCache()
go fnTokenCache.StartGC(context.Background(), time.Second*10)

client := sdk.NewClientWithOpts(
    gatewayUrl,
    httpClient, sdk.WithAuthentication(auth),
    sdk.WithFunctionTokenCache(fnTokenCache),
)
```

Reply:
Is that example in the examples in the README?

How would you change the example so someone could stop the StartGC?

Reply:
Yes this example is also README.

Instead of passing the background context in the example it can be changed to use a context that can be cancelled.